AlfrescoApp/View Controllers/FileFolder View Controller/FileFolderCollectionViewController.m

```diff
@@ -189,12 +189,12 @@ - (void)viewDidLoad
 
     self.collectionView.dataSource = self;
     self.collectionView.delegate = self;
-    self.listLayout = [BaseCollectionViewFlowLayout new];
-    self.listLayout.itemHeight = kCellHeight;
+    self.listLayout = [[BaseCollectionViewFlowLayout alloc] initWithNumberOfColumns:1 itemHeight:kCellHeight shouldSwipeToDelete:YES];
     self.listLayout.dataSourceInfoDelegate = self;
+    self.gridLayout = [[BaseCollectionViewFlowLayout alloc] initWithNumberOfColumns:3 itemHeight:kCellHeight shouldSwipeToDelete:NO];
```

Comment:
OK, just seen this so I can see why you've got the long init method above.

Perhaps it might be better defining an enum with a "list view mode" and a "grid view mode"..?